lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -41,6 +41,7 @@
 from fastapi_utils.inferring_router import InferringRouter
 from pydantic import ValidationError
 from pydantic.main import BaseModel
+from routes import request_config
```

Comment:
```suggestion
from routes import (
    Mapper,
    request_config,
)
```

Reply:
I still don't understand why we're doing runtime imports instead of placing this under `if TYPE_CHECKING:`

Reply:
I am not against placing them under `if TYPE_CHECKING:`, but I also don't see that providing much of an advantage where there is no risk of a circular import or of adding unnecessary dependencies to a package (which is not the case here).